packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -3040,6 +3050,14 @@ function remountFiber(
     } else {
       returnFiber.firstEffect = returnFiber.lastEffect = current;
     }
+    const deletions = returnFiber.deletions;
+    if (deletions === null) {
+      returnFiber.deletions = [current];
+    } else {
+      deletions.push(current);
+    }
+    // TODO (effects) Rename this to better reflect its new usage (e.g. ChildDeletions)
+    returnFiber.effectTag |= Deletion;
```

Comment:
Same here. Can remove the flag from the child.

Reply:
Whoops, yup.